Template10 (Library)/Controls/HamburgerMenu.xaml.cs

```diff
@@ -502,9 +502,11 @@ public void UpdatePaneMarginToShowHamburgerButton()
             }
         }
 
-        private async Task UpdateSelectedAsync(HamburgerButtonInfo previous, HamburgerButtonInfo value)
+        private bool IsCanceledNavigation;
```

Comment:
I will say I regret you did these changes in this PR. Makes it more difficult to review with so many changes only being renamed variables/args.